### DISCUSSION TITLE
Are Azure still supported for 4.12? No azure build found

### DISCUSSION THREAD

**Discussion by jryberg** [okd-project/okd#1562] [Q&A]
Hi,

I'm looking to install okd in Azure but the installer cannot download images

`time="2023-04-14T09:10:47+02:00" level=fatal msg="failed to fetch Metadata: failed to fetch dependency of \"Metadata\": failed to fetch dependency of \"Bootstrap Ignition Config\": failed to fetch dependency of \"CVO Ignore\": failed to fetch dependency of \"Openshift Manifests\": failed to generate asset \"Image\": stable/x86_64: No azure build found"`

I'm using this installer: https://github.com/okd-project/okd/releases/download/4.12.0-0.okd-2023-04-01-051724/openshift-install-linux-4.12.0-0.okd-2023-04-01-051724.tar.gz

It's almost exactly like this Q&A: https://github.com/okd-project/okd/discussions/1186 but another release.

Is this a known issue? I'm happy to assist if needed with any testing.

**Comment by vrutkovs** [comment]
See https://github.com/okd-project/okd/issues/13#issuecomment-564692550
